etc/profile-m-z/strawberry.profile

```diff
@@ -0,0 +1,49 @@
+# Firejail profile for strawberry
+# Description: A music player and music collection organizer
+# This file is overwritten after every install/update
+# Persistent local customizations
+include strawberry.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/strawberry
+noblacklist ${HOME}/.config/strawberry
+noblacklist ${HOME}/.local/share/strawberry
+noblacklist ${MUSIC}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc 
+
+apparmor
+caps.drop all
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+# blacklisting of ioprio_set system calls breaks strawberry
+seccomp !ioprio_set
+shell none
+tracelog
+
+private-bin strawberry,strawberry-tagreader
+private-cache
+private-dev
+private-etc ca-certificates,crypto-policies,fonts,host.conf,hostname,hosts,nsswitch.conf,pki,resolv.conf,ssl
+disable-mnt
```

Comment:
Move disable-mnt at the top of this block, just above `private-bin`,

Reply:
I've checked wruc and everything was fine, should I now add `include whitelist-runuser-common.inc` to strawberry profile? What happens to `mutter-Xwaylandauth.*` if I do it this way?

Reply:
> I've checked wruc and everything was fine, should I now add include whitelist-runuser-common.inc to strawberry profile?

If it works, yes.

> What happens to mutter-Xwaylandauth.* if I do it this way?

Globbing in `whitelist` paths is not supported in 0.9.62, that's why I sayed you should remove it.

Reply:
OK, I thought it breaks something, well it's resolved then.